### PR TITLE
Use go fmt

### PR DESCRIPTION
**Pull request by nkprince007**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ go:
 
 before_install:
   - go get -v
+  - go fmt
```
Comment (adtac) [inline .travis.yml]: `go fmt .` (notice the dot) will list the files that have unformatted content. With that you can check if the code is formatted properly by checking if the output is empty.

If only we had a static code analysis tool that does all this automatically ;) Wanna add a coafile instead?

Comment (nkprince007) [reply .travis.yml]: If it is supported why not.

**Comment by nkprince007** [comment]
@coafile send in a pull request

**Comment by nkprince007** [comment]
@adtac done, the CI is ready with coala. 🎉 😄

**Comment by adtac** [comment]
Thanks! Merged it :)

**Pull request merged**